helper/helper.go

```diff
@@ -0,0 +1,34 @@
+package helper
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+const (
+	EnvVar = "ASPNETCORE_URLS"
+)
+
```

Comment:
Can you add a comment about what this is doing for context!

Reply:
done

Reply:
Sorry, I rather confusingly only highlighted this part, but I meant for the entire file (now cmd/port-chooser/internal/port-chooser.go`. Just a one liner explaining the port selection would be sufficient.